normits_demand/models/forecasting/EDGE_growth/run.py

```diff
@@ -0,0 +1,315 @@
+            if to_home:
+                zonal_base_demand_mx = utils.transpose_matrix(zonal_base_demand_mx)
+            (
+                np_stn2stn_base_demand_mx,
+                zonal_from_to_stns,
+            ) = utils.zonal_from_to_stations_demand(
+                zonal_base_demand_mx,
+                irsj_probs_segment,
+                model_stations_tlcs_len,
+                userclass,
+                time_period
+            )
+            # store matrix total demand
+            tot_input_demand = round(np_stn2stn_base_demand_mx.sum())
+            # add to overall base demand
+            overall_base_demand += tot_input_demand
+            # apply growth
+            np_stn2stn_grown_demand_mx = (
+                apply_growth.apply_demand_growth(
+                    np_stn2stn_base_demand_mx,
+                    splitting_matrices[purpose],
+                    filled_growth_matrices[purpose],
+                    len(np_stn2stn_base_demand_mx),
+                    growth_method,
+                    to_home,
+                )
+            )
+            # get movements where no growth has been applied
+            no_growth_movements = np_stn2stn_base_demand_mx[
+                np_stn2stn_base_demand_mx == np_stn2stn_grown_demand_mx
+            ]
+            # Add to the total not grown demand
+            overall_not_grown_demand = (
+                overall_not_grown_demand + no_growth_movements.sum()
+            )
+            # store matrix total demand
+            tot_output_demand = round(np_stn2stn_grown_demand_mx.sum())
+            # append to growth summary df
+            # empty dataframe for growth summary
+            segment_growth_summary = pd.DataFrame(
+                {
+                    "Time_Period": [time_period],
+                    "Demand_Segment": [segment],
+                    "Base_Demand": [tot_input_demand],
+                    f"{forecast_year}_Demand": [tot_output_demand],
+                }
+            )
+            growth_summary = pd.concat(
+                [growth_summary, segment_growth_summary], axis=0
+            )
+            # convert back to zonal level demand
+            zonal_grown_demand_mx = (
+                utils.convert_stns_to_zonal_demand(
+                    np_stn2stn_grown_demand_mx,
+                    zonal_from_to_stns,
+                    time_period,
+                    to_home,
+                )
+            )
+            # add to grown matrices dictionary
+            factored_matrices[segment] = zonal_grown_demand_mx
+
+    return loading.TPLoopOutput(overall_not_grown_demand, overall_base_demand, factored_matrices, growth_summary)
+
+
+def run_edge_growth(params: forecast_cnfg.EDGEParameters) -> None:
+    """Run Growth Process."""
+    LOG.info("#" * 80)
+    LOG.info(
+        "Started Process @ %s",
+        timing.get_datetime("%d-%m-%Y  %H:%M:%S"),
+    )
+    LOG.info("#" * 80)
```

Comment:
```suggestion
    LOG.info("Started applying EDGE Growth")
    LOG.info("#" * 80)
```

We don't need the datetime being stored as it is part of all the log messages anyway